### DISCUSSION TITLE
field override based on custom (key:value) variable

### DISCUSSION THREAD

**Discussion by IlyaDeskov** [grafana/grafana#65151] [Dashboards, Panels & Visualizations]
We have a custom variable where we can set a key:value mapping to override fields in one place for the whole dashboard. We just need to add a new value mapping type:
Value
Range
regular expression
Special
**custom variable**
It seems to be very convenient.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is feature request is fairly non specific. If this is something youre looking for, please make an [issue](https://github.com/grafana/grafana/issues) with what you'd like added, with some discrete examples of each kind of mapping type, so we can better figure out what a priority on the feature would be. Thank you!
